### PR TITLE
Add `actor` property to WorkflowRun

### PR DESCRIPTION
**Pull request by gbhand**
The GitHub API [returns an `actor` field when listing workflow runs](https://docs.github.com/en/free-pro-team@latest/rest/actions/workflow-runs?apiVersion=2022-11-28#list-workflow-runs-for-a-workflow) but `WorkflowRun` does not include `actor`. 

This PR adds the `actor` property of `NamedUser` type.

Not sure about field ordering conventions so just followed the API response schema.

### CODE REVIEW

File: github/WorkflowRun.py [hunk 1]
```diff
@@ -318,6 +325,8 @@ def _useAttributes(self, attributes: dict[str, Any]) -> None:
             self._created_at = self._makeDatetimeAttribute(attributes["created_at"])
         if "updated_at" in attributes:  # pragma no branch
             self._updated_at = self._makeDatetimeAttribute(attributes["updated_at"])
+        if "actor" in attributes:  # pragma no branch
+            self._actor = self._makeClassAttribute(github.NamedUser.NamedUser, attributes["actor"])
```
Comment (trim21) [inline github/WorkflowRun.py:329]: Need to import this class I guess

Comment (gbhand) [reply github/WorkflowRun.py:329]: Yeah wasn't too sure what is needed here. I saw later on `github.Repository.Repository` was referenced but only imported by name with `TYPE_CHECKING`.

Would adding `import github.NamedUser` be the right move?

Comment (trim21) [reply github/WorkflowRun.py:329]: yes

Comment (gbhand) [reply github/WorkflowRun.py:330]: Addressed [841daed](https://github.com/PyGithub/PyGithub/pull/2764/commits/841daed3e18a5db15e9cdbc153ec505fb10d8147)

**Comment by trim21** [comment]
lgtm

**Review by EnricoMi — CHANGES_REQUESTED** [review]
Can we add this new attribute to https://github.com/PyGithub/PyGithub/blob/main/tests/WorkflowRun.py#L40? LGTM besides this.

**Comment by EnricoMi** [comment]
@gbhand?

**Comment by gbhand** [comment]
Apologies @EnricoMi, haven't checked this in quite a while. Added to the unit test in [6347cb2](https://github.com/PyGithub/PyGithub/pull/2764/commits/6347cb221bb28f1a1b4bbb6b12bd0de8e8cd5d30).

**Review by EnricoMi — APPROVED** [review]
LGTM!

**Pull request merged**